### PR TITLE
Doodle

### PR DESCRIPTION
**Pull request by romanpeters**
### Keyword functionality
Keywords can be used to trigger functions, just like commands can.
The main differences are:
- No special character needed to trigger
- Keywords can be anywhere within the message text, even within words in the text

Since keywords triggering things can be very annoying, this feature should generally not be used. Use commands instead.

### New database table
The table `database.Doodle` can be used to persistently store Doodle polls.

### Doodle Saving
When a URL to a Doodle poll is posted (keyword: "https://doodle.com/poll/"), this URL is saved to the database.

### Doodle Retrieving
The command `!doodle` returns the last Doodle poll URL saved to the database.

### CODE REVIEW

**Comment by romanpeters** [comment]
Python Black keeps crashing on me, so if someone else could Black this before merging, that would be great.

File: devbot/database.py
Comment (RobinSikkens) [inline devbot/database.py]: Please add `.` to your docstrings

File: devbot/commands/doodle.py
Comment (RobinSikkens) [inline devbot/commands/doodle.py]: Please add `.` to your docstring

File: devbot/commands/doodle.py
Comment (RobinSikkens) [inline devbot/commands/doodle.py]: Please add `.` to your docstring

File: devbot/registry.py
Comment (RobinSikkens) [inline devbot/registry.py]: Please add `.` to your docstrings

File: devbot/bot.py
Comment (RobinSikkens) [inline devbot/bot.py]: Please elaborate in this info message, also should this INFO or DEBUG?

**Review by RobinSikkens — CHANGES_REQUESTED** [review]

File: devbot/bot.py
Comment (romanpeters) [reply devbot/bot.py]: It mirrors [line 54](https://github.com/DevOps-Utrecht/bot/blob/f409e1c8a8af1321286469e20bf61e9fd698b4fc/devbot/bot.py#L54). Should I convert both to DEBUG?

Comment (RobinSikkens) [reply devbot/bot.py]: It's okay for it to be info if you could change them both to be more descriptive however that would be great.

**Review by RobinSikkens — DISMISSED** [review]

File: devbot/bot.py
Comment (RobinSikkens) [inline devbot/bot.py]:
```suggestion
            LOGGER.info(f"Keyword: {keyword} was triggered.")
```

File: devbot/database.py
Comment (RobinSikkens) [inline devbot/database.py]: Using id shadows a function, this might cause issues. Check the [docs](https://docs.python.org/3/library/functions.html#id) for more info.

File: devbot/bot.py
Comment (romanpeters) [reply devbot/bot.py]: Huh, I thought I had fixed that

File: devbot/database.py
Comment (romanpeters) [reply devbot/database.py]: Good point

**Review by RobinSikkens — DISMISSED** [review]

**Comment by romanpeters** [comment]
I can't merge it without a new approving review, but feel free to do so. Should be all good.

**Review by RobinSikkens — APPROVED** [review]

**Pull request merged**
